packages/app/src/app/pages/Dashboard/Components/SyncedSandbox/SyncedSandboxCard.tsx

```diff
@@ -36,19 +28,12 @@ export const SyncedSandboxCard = ({ name, path, url, ...props }) => {
                 <Link
                   to={url}
                   css={{
+                    display: 'flex',
+                    overflow: 'hidden',
                     textDecoration: 'none',
                   }}
                 >
```

Comment:
We might want a wrapper around the `react-router` `Link` component to ensure we know exactly what we're rendering. It feels weird to have `css` added straight to the `Link`. I know I started, with the `textDecoration` haha, but we might want to tackle this in the near future.

Reply:
Right, but in this case `textDecoration` feels like a global reset (maybe it should be considered in the future) while the others are related to the use of the Link as an element wrapping something else.

Reply:
let's consider the Link in prism first, combined with the new typography components